src/accelerate/utils/parallel_state.py

```diff
@@ -0,0 +1,790 @@
+# Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
```

Comment:
I think we don't need this file. Just import the `parallel_state` from `megatron.core`.

```python
from accelerate.utils.imports import is_megatron_lm_available

if is_megatron_lm_available():
    from accelerate.utils.megatron_lm import mpu
```